annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -166,7 +168,7 @@ public AnnotatedRecord convertResponseToAnnotatedRecord(boolean replace) {
                 resolveVariantClassification(),
                 resolveVariantType(),
                 resolveReferenceAllele(),
-                resolveReferenceAllele(),
+                mRecord.getTUMOR_SEQ_ALLELE1(),
```

Comment:
After discussing with @inodb we decided to make `Tumor_Seq_Allele1` immutable. This will have no consequence on how the variant gets annotated since we are adding logic to handle ambiguous tumor seq allele cases (from `cbioportal/MafUtil`)